app/controllers/design_methods_controller.rb

```diff
@@ -31,6 +32,7 @@ def new
   # - @design_method: the design method to be edited
   def edit
     @design_method = DesignMethod.find(params[:id])
+    puts @design_method.references
```

Comment:
@thuangy I think this is leftover from debugging - if so can you remove? Thanks.